examples/produce_part_sweep/README.md

```
@@ -0,0 +1,35 @@
+```shell
+pc inspect dxf
+pc inspect cylinder
+pc inspect clock
+```
+
+
+## Parts
+
+### clock
+<table><tr>
+<td valign=top><a href="clock.sweep"><img src="././clock.svg" style="width: auto; height: auto; max-width: 200px; max-height: 200px;"></a></td>
```

Markdownlint (0.37.0)

Element: td
Inline HTML

(MD033, no-inline-html)